File: src/main/java/no/fintlabs/dispatch/repository/InstanceReceiptDispatchRepository.java
package no.fintlabs.dispatch.repository;

import no.fintlabs.dispatch.model.InstanceReceiptDispatchEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface InstanceReceiptDispatchRepository extends JpaRepository<InstanceReceiptDispatchEntity, String> {
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/infrastructure/dispatch/repository/InstanceReceiptDispatchRepository.kt
package no.novari.flyt.egrunnerverv.gateway.infrastructure.dispatch.repository

import no.novari.flyt.egrunnerverv.gateway.infrastructure.dispatch.model.InstanceReceiptDispatchEntity
import org.springframework.data.jpa.repository.JpaRepository

interface InstanceReceiptDispatchRepository : JpaRepository<InstanceReceiptDispatchEntity, String>
